readthedocs/organizations/models.py

```diff
@@ -215,10 +218,7 @@ def get_absolute_url(self):
         )
 
     def __str__(self):
-        return '{organization}/{team}'.format(
-            organization=self.organization.name,
-            team=self.name,
-        )
+        return self.name
```

Comment:
When an organization is deleted, the historical record from its teams are still there and an invalid organization pk is held, any time we want to access to `team_record.organization` it will error. When checking these models on the terminal, they will error (__str__ is called), so I'm just printing the name now.

Reply:
This seems bad. This is changing the UX for the actual site I think, which we don't want to do. Is there not a way to limit this code in the historical model, instead of changing the functioning of the actual code?

Reply:
this change is only when printing things on the console, for the templates we access to the organization itself

https://github.com/readthedocs/readthedocs-corporate/blob/6ae87d271fb23262df46718d184469d134137b10/readthedocsinc/projects/forms.py#L103-L108

Which aren't a record model, so the object always has an organization. We can also add a try/except block, but since this only affects the shell not sure.